src/Uno.UI.Wasm/WasmScripts/Uno.UI.js

```diff
@@ -971,6 +971,9 @@ var Uno;
                 const shouldRaiseLoadEvents = WindowManager.isLoadEventsEnabled
                     && this.getIsConnectedToRootElement(childElement);
                 parentElement.removeChild(childElement);
+                // Mark the element as unarranged, so if it gets measured while being
```

Comment:
why this file?

Reply:
I didn't modify it so I thought it might be important